common/src/androidTest/kotlin/io/homeassistant/companion/android/JacksonDummyTest.kt

```diff
@@ -0,0 +1,36 @@
+package io.homeassistant.companion.android
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.junit.Assert.assertEquals
+import org.junit.Test
```

Comment:
FYI we can't use junit5 on instrumentationTest since junit5 needs API 26 minimum due to the JAVA_VERSION it uses.